Specs/Scene/GroundPrimitiveSpec.js

```diff
@@ -626,8 +634,11 @@ defineSuite([
             classificationType : ClassificationType.TERRAIN
         });
 
-        scene.morphToColumbusView(0);
```

Comment:
@likangning93 Was this supposed to be here?  This scene wasn't being tested at all, and the test failed when i did `largeScene.morphToColumbusView`

Reply:
This must have been a mistake, sorry about that!